packages/project-access/src/project/cap.ts

```diff
@@ -362,7 +369,12 @@ async function loadCdsModuleFromProject(capProjectPath: string): Promise<CdsFaca
             `Could not load cds module. Attempt to load module @sap/cds from project threw error '${loadProjectError}', attempt to load module @sap/cds from @sap/cds-dk threw error '${loadError}'`
         );
     }
-    return 'default' in module ? module.default : module;
+    const cds = 'default' in module ? module.default : module;
+    // Fix when switching cds versions dynamically
+    if (global) {
+        global.cds = cds;
```

Comment:
Switching global variables doesn't seem to fix all the cases, since someone else could load a different version in async code and I suspect we'll end up with having the same issue.

Reply:
@voicis, good point, but how else can we do it? 
First idea was to add `global.cds = cds` in CAP-related functions in [packages/project-access/src/project/cap.ts](https://github.com/SAP/open-ux-tools/blob/main/packages/project-access/src/project/cap.ts), like `getCapModelAndServices()`, `getCapEnvironment()`, right before we do a call on cds instance. However, all those functions call `loadCdsModuleFromProject()` to get a cds instance, so `global.cds = cds` is done every time before we call functions on cds. Any other idea?

Reply:
I looked at the relevant CDS coding and there is nothing else we can do in this case.

Since loading CDS module is not exposed and we do not keep the CDS module reference, but always load it this fix should be sufficient.